Distiller-server/src/main/java/com/mapr/distiller/server/recordtypes/ThreadResourceRecord.java

```diff
@@ -84,78 +141,43 @@ public static boolean produceRecord(RecordQueue thread_resources, String produce
 	public String toString(){
```

Comment:
Yeah @Override is a compiler instruction. It is basically used whenever we try to override the implementation of the parent class method in the child class. In case of toString(), I think it is part of class Object and all classes inherently extend Object class.

http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html